forge-game/src/main/java/forge/game/trigger/Trigger.java

```diff
@@ -613,4 +613,10 @@ public SpellAbility ensureAbility(final IHasSVars sVarHolder) {
     public SpellAbility ensureAbility() {
         return ensureAbility(this);
     }
+
+    @Override
+    public void setOverridingAbility(SpellAbility overridingAbility0) {
+        super.setOverridingAbility(overridingAbility0);
+        overridingAbility0.setTrigger(this);
```

Comment:
It kind of makes sense but I'm wondering if it could cause problems here:
https://github.com/Card-Forge/forge/blob/1186d4ee221730f1d6b12f60bde74e84b3cef6ba/forge-game/src/main/java/forge/game/card/ActivationTable.java#L27

I didn't think of a good test yet but does the root now lead to the copied Trigger so it can't find the original?

Reply:
imo that shouldn't have any effect i think

Trigger.setOverridingAbility is only called when creating the trigger

for example: Tested with `Omnath, Locus of Creation` and `Strionic Resonator` the ability still works as it should